### PR TITLE
Refactor FXIOS-9823 Use StandardImageIdentifiers in app extensions

### PR DESCRIPTION
**Pull request by thatswinnie**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-9823)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/21574)

## :bulb: Description
Replaces all hard-coded image strings that can be found in the Acorn image library with `StandardImageIdentifiers`.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ import Foundation
 public struct StandardImageIdentifiers {
     // Icon size 16x16
     public struct Small {
+        public static let externalLink = "externalLinkSmall"
```
Comment (cyndichin) [inline BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift:13]: I notice that this doesn't match what we have the figma file [here](https://www.figma.com/design/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?node-id=0-1&t=41X6bB7npySvuD87-0). It does match the github repo, which I assume should be the [source of truth](https://github.com/FirefoxUX/acorn-icons/tree/main/icons/mobile/16). 

@cwzilla should we also update the Figma File or should we point to the github repo going forward?

**Review by cyndichin — APPROVED** [review]
thanks for making the changes in multiple places, looks great! left one comment for clarity.

File: BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift (hunk 1 above)
Comment (cyndichin) [reply BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift:13]: @cwzilla sorry I wasn't clear. It looks like its in the small category, but in the figma file I see it under the large category instead. Should it be moved to 16x16? Also let me know if I'm misreading the file!

Comment (cwzilla) [reply BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift:13]: @cyndichin oh gotcha. just did some digging and it looks like we have 2 variations of the external link icon, so what's in code/repo is correct, the 16x16 icon is just missing in Figma. I'll update it :)

Comment (cyndichin) [reply BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift:13]: thanks!

**Pull request merged**